content/bn/kubernetes.md

```diff
@@ -0,0 +1,28 @@
+---
+title: কুবারনেটিস
+status: Feedback Appreciated
+category: প্রযুক্তি
+---
+
+## এটা কি
+
+কুবারনেটস (Kubernetes), প্রায়ই K8s হিসাবে সংক্ষিপ্ত হয়, একটি ওপেন সোর্স কন্টেইনার অর্কেস্ট্রেটর যা আধুনিক পরিকাঠামোতে [কন্টেইনারাইজড](/container/) অ্যাপ্লিকেশনের জীবনচক্রকে স্বয়ংক্রিয় করে। এটি একটি ডেটা সেন্টারের অপারেটিং সিস্টেমের মতো, যা অংশ-বিভাগিত সিস্টেম (Distributed System) জুড়ে চলমান অ্যাপ্লিকেশনগুলিকে পরিচালনা করে (ঠিক যেমন আপনার ল্যাপটপের OS আপনার অ্যাপগুলি পরিচালনা করে) ‍।
+
+কুবারনেটস একটি [ক্লাস্টারের](/cluster/) নোড জুড়ে কোনটেনারের সময়সূচী নির্ধারণ করে। এটি বেশ কয়েকটি পরিকাঠামো নির্মাণ, যাকে কখনও কখনও "প্রিমিটিভস" হিসাবে উল্লেখ করা হয়, যেমন একটি অ্যাপের দৃষ্টান্ত, লোড ব্যালেন্সার, পারসিষটেন্ট স্টোরেজ এবং অন্যান্যগুলিকে এমনভাবে একত্রিত করে যাতে সেগুলি অ্যাপ্লিকেশনগুলিতে তৈরি করা যায়।
+
+কুবারনেটস অটোমেশন এবং এক্সটেনসিবিলিটি সক্ষম করে, ব্যবহারকারীদেরকে পুনর্গঠনযোগ্য ও ঘোষণামূলকভাবে অ্যাপ্লিকেশন স্থাপন করতে দেয়। কুবারনেটস ইকোসিস্টেমের সফ্টওয়্যার প্রডাক্ট এবং প্রজেক্টগুলি কুবারনেটস [এপিআই](/application-programming-interface/) প্রসারিত করার জন্য সেই অটোমেশন এবং এক্সটেনসিবিলিটির সুবিধা নেয়। এটি তাদের কুবারনেটসের অটোমেশনকে উদ্দেশ্যসাধনের জন্য ব্যবহার করে এবং তাদের তুলসগুলিকে অভিজ্ঞ কুবারনেটস অনুশীলনকারীদের কাছে আরও অ্যাক্সেসযোগ্য করে তুলতে সক্ষম করে।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+ইনফ্রাস্ট্রাকচার অটোমেশন এবং ঘোষণামূলক কনফিগারেশন ম্যানেজমেন্ট দীর্ঘকাল ধরে একটি গুরুত্বপূর্ণ ধারণা এবং [ক্লাউড কম্পিউটিং](/cloud-computing/) জনপ্রিয়তা অর্জন করায় এটি আরও বেশি চাপে পড়েছে।
```

Comment:
Kindly use English form in Brackets  for words like these`ইনফ্রাস্ট্রাকচার অটোমেশন`